tests/TestCase/Model/Behavior/UploadBehaviorTest.php

```diff
@@ -763,4 +798,18 @@ public function testNameCallback()
 
         $this->assertEquals($expected, $behavior->constructedFiles);
     }
+
+    private function transformUploadedFilesToArray(array $data): array
+    {
+        return array_map(
+            fn(UploadedFileInterface $file) => [
+                'tmp_name' => '',
+                'error' => $file->getError(),
+                'name' => $file->getClientFilename(),
+                'type' => $file->getClientMediaType(),
+                'size' => $file->getSize(),
+            ],
+            $data
+        );
+    }
```

Comment:
>  Parse error: syntax error, unexpected '$file' (T_VARIABLE), expecting ',' or ')' in /home/runner/work/cakephp-upload/cakephp-upload/tests/TestCase/Model/Behavior/UploadBehaviorTest.php on line 805

To fix the above error: 

```suggestion
        return array_map(
            function (UploadedFileInterface $file) {
                return [
                    'tmp_name' => '',
                    'error' => $file->getError(),
                    'name' => $file->getClientFilename(),
                    'type' => $file->getClientMediaType(),
                    'size' => $file->getSize(),
                ];
            },
            $data
        );
    }
```